api/index.js

```diff
@@ -79,7 +79,7 @@ const executeRequest = ({ url, ...options }) =>
     .then(processStatus)
     .then(response => processBody(response, { ...options }))
 
-const performRequest = (url, options) => {
+const performRequest = async (url, options) => {
```

Comment:
No need in that. `async` basically converts function to Promise. If you return Promise, there is no need to call it async.